client/src/AppBundle/Resources/views/Report/Report/declaration.html.twig

```diff
@@ -72,7 +76,8 @@
                 'items': [
                     {},
                     {},
-                    {'dataTarget': 'agreed-behalf-deputy-explanation-section' }
```

Comment:
Now this is removed, you can take out the `<div id="agreed-behalf-deputy-explanation-section">...</div>` block below. Symfony can be a bit weird if you render the same form field twice.

Reply:
Good point